webapps/src/main/webapp/vue-app/tasks-management/components/ProjectTasks/TaskViewListItem.vue

```diff
@@ -38,13 +38,16 @@
         @click="openTaskDrawer()">{{ task.task.title }}
       </a>
     </div>
-    <div class="taskAssignee d-flex v-avatar flex-nowrap me-2 justify-start">
+    <div
+        class="taskAssignee d-flex v-avatar flex-nowrap me-2 justify-start position-relative"
+        :style="`min-height:${avatarSize+4}px`">
```

Comment:
please run `npm run lint` before committing and ensure the warnings numbers doesn't increase

```suggestion
      class="taskAssignee d-flex v-avatar flex-nowrap me-2 justify-start position-relative"
      :style="`min-height:${avatarSize+4}px`">
```

Reply:
done